content/2017-09-06-authenticating-against-the-composable-api.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Authenticating against the Composable API
+title: "Blog"
```

Comment:
this looks wrong

Reply:
Yes, I realized it after pushing and it's fixed now.